tests/queries/0_stateless/01158_zookeeper_log.reference

```diff
@@ -0,0 +1,40 @@
+log
+Receive	0	Watch	/test/01158/default/rmt/log	0	0	0	0	0	ZOK	CHILD	CONNECTED		0	0	0	0
+Send	0	Create	/test/01158/default/rmt/log	0	0	0	0	4	\N	NONE	NONE		0	0	0	0
+Receive	0	Create	/test/01158/default/rmt/log	0	0	0	0	4	ZOK	NONE	NONE	/test/01158/default/rmt/log	0	0	0	0
+Send	0	Create	/test/01158/default/rmt/log/log-	0	1	0	0	1	\N	NONE	NONE		0	0	0	0
+Receive	0	Create	/test/01158/default/rmt/log/log-	0	1	0	0	1	ZOK	NONE	NONE	/test/01158/default/rmt/log/log-0000000000	0	0	0	0
+parts
+Send	0	Multi		0	0	0	5	0	\N	NONE	NONE		0	0	0	0
+Send	0	Create	/test/01158/default/rmt/log/log-	0	1	0	0	1	\N	NONE	NONE		0	0	0	0
+Send	0	Remove	/test/01158/default/rmt/block_numbers/all/block-0000000000	0	0	-1	0	2	\N	NONE	NONE		0	0	0	0
+Send	0	Remove	/test/01158/default/rmt/temp/abandonable_lock-0000000000	0	0	-1	0	3	\N	NONE	NONE		0	0	0	0
+Send	0	Create	/test/01158/default/rmt/blocks/all_6308706741995381342_2495791770474910886	0	0	0	0	4	\N	NONE	NONE		0	0	0	0
+Send	0	Create	/test/01158/default/rmt/replicas/1/parts/all_0_0_0	0	0	0	0	5	\N	NONE	NONE		0	0	0	0
+Receive	0	Multi		0	0	0	5	0	ZOK	NONE	NONE		0	0	0	0
+Receive	0	Create	/test/01158/default/rmt/log/log-	0	1	0	0	1	ZOK	NONE	NONE	/test/01158/default/rmt/log/log-0000000000	0	0	0	0
+Receive	0	Remove	/test/01158/default/rmt/block_numbers/all/block-0000000000	0	0	-1	0	2	ZOK	NONE	NONE		0	0	0	0
+Receive	0	Remove	/test/01158/default/rmt/temp/abandonable_lock-0000000000	0	0	-1	0	3	ZOK	NONE	NONE		0	0	0	0
+Receive	0	Create	/test/01158/default/rmt/blocks/all_6308706741995381342_2495791770474910886	0	0	0	0	4	ZOK	NONE	NONE	/test/01158/default/rmt/blocks/all_6308706741995381342_2495791770474910886	0	0	0	0
+Receive	0	Create	/test/01158/default/rmt/replicas/1/parts/all_0_0_0	0	0	0	0	5	ZOK	NONE	NONE	/test/01158/default/rmt/replicas/1/parts/all_0_0_0	0	0	0	0
+Send	0	Exists	/test/01158/default/rmt/replicas/1/parts/all_0_0_0	0	0	0	0	0	\N	NONE	NONE		0	0	0	0
+Receive	0	Exists	/test/01158/default/rmt/replicas/1/parts/all_0_0_0	0	0	0	0	0	ZOK	NONE	NONE		0	0	96	0
+blocks
+Send	0	Multi		0	0	0	3	0	\N	NONE	NONE		0	0	0	0
+Send	0	Create	/test/01158/default/rmt/blocks/all_6308706741995381342_2495791770474910886	0	0	0	0	1	\N	NONE	NONE		0	0	0	0
+Send	0	Remove	/test/01158/default/rmt/blocks/all_6308706741995381342_2495791770474910886	0	0	-1	0	2	\N	NONE	NONE		0	0	0	0
+Send	0	Create	/test/01158/default/rmt/temp/abandonable_lock-	1	1	0	0	3	\N	NONE	NONE		0	0	0	0
+Receive	0	Multi		0	0	0	3	0	ZOK	NONE	NONE		0	0	0	0
+Receive	0	Create	/test/01158/default/rmt/blocks/all_6308706741995381342_2495791770474910886	0	0	0	0	1	ZOK	NONE	NONE	/test/01158/default/rmt/blocks/all_6308706741995381342_2495791770474910886	0	0	0	0
+Receive	0	Remove	/test/01158/default/rmt/blocks/all_6308706741995381342_2495791770474910886	0	0	-1	0	2	ZOK	NONE	NONE		0	0	0	0
+Receive	0	Create	/test/01158/default/rmt/temp/abandonable_lock-	1	1	0	0	3	ZOK	NONE	NONE	/test/01158/default/rmt/temp/abandonable_lock-0000000000	0	0	0	0
```

Comment:
What is `NONE` value maybe `NULL` or just `0`?

Reply:
Will make it Nullable